js/fc.js

```diff
@@ -604,6 +600,68 @@ var FC = {
             'Disabled'
         ];
     },
+    getRxTypes: function() {
+        // Keep value field in sync with rxReceiverType_e in rx.h
+        var rxTypes = [
+            {
+                name: 'RX_SERIAL',
+                bit: 3,
+                value: 3,
+            },
+            {
+                name: 'RX_PPM',
+                bit: 0,
+                value: 1,
```

Comment:
PWM and PPM are mixed up:
```
typedef enum {
    RX_TYPE_NONE        = 0,
    RX_TYPE_PWM         = 1,
    RX_TYPE_PPM         = 2,
    RX_TYPE_SERIAL      = 3,
    RX_TYPE_MSP         = 4,
    RX_TYPE_SPI         = 5,
} rxReceiverType_e;
```

Reply:
I think those are incorrectly named in the configurator code. There's `RX_PPM` and `RX_PARALLEL_PWM` and those are the same thing! Shouldn't it be `RX_PPM` and `RX_PWM`?

Reply:
Agree, let's rename them to RX_PPM and RX_PWM